addOns/diff/src/main/javahelp/org/zaproxy/zap/extension/diff/resources/help/contents/diff.html

```diff
@@ -9,10 +9,17 @@
 <BODY>
 <H1>Diff</H1>
 <p>
-	The Diff add-on allows you to compare 2 requests or responses.
+	The Diff add-on allows you to compare 2 messages (requests or responses).
 </p>
 <p>
-	It is typically accessed via right click menu items.
+	It can be accessed by selecting two messages from any view component that shows more than one message e.g. the Sites tree and then opening the context menu (usually done by pressing the right-click button). You'll see the options "Compare 2 Requests", "Compare 2 Responses" there clicking one of it will open the diff window. 
+
```

Comment:
Close/open paragraph.

Reply:
Maybe br's instead of paragraphs, the idea is to fix the images cutting the text.